python/cuml/tsa/arima.pyx

```diff
@@ -251,8 +239,8 @@ class ARIMA(Base):
             raise ValueError("ERROR: Invalid order. Required: p,q,P,Q <= 4")
 
         # Get device array. Float64 only for now.
-        self.d_y, _, self.n_obs, self.batch_size, self.dtype \
-            = input_to_dev_array(y, check_dtype=np.float64)
+        self.d_y, self.n_obs, self.batch_size, self.dtype \
```

Comment:
```suggestion
        self._d_y, self.n_obs, self.batch_size, self.dtype \
```

If I'm not mistaken, at least `d_y` needs to be renamed with the leading `_` to be able to inherit the format conversion behavior from the base class and behave like the other (cumlarray updated) estimators